pkg/registry/scripts/kube-client.js

```diff
@@ -952,6 +953,18 @@
                 }
             });
 
+            /* The host filter */
+            registerFilter({
+                name: "host",
+                digest: function(arg) {
+                    if (typeof arg === "string")
+                        return arg;
+
+                    var spec = arg.spec;
+                    return spec ? spec.nodeName : null;
```

Comment:
So strange that the nodeName is set on the spec and not the status. Although in the previous code that was object.status.spec.nodeName. I guess that was incorrect after some kubernetes API changes?